org_status/export/Exporter.py

```diff
@@ -0,0 +1,15 @@
+class Exporter:
```

Comment:
Rename the module to `encoders`, call this class `OrgListEncoder` and move this file's contents to `__init__.py`.